src/SiteInformation/shared/EditLocationModal/EditLocationModal.js

```diff
@@ -199,7 +199,6 @@ const EditLocationModal = ({ action }) => {
       aria-labelledby="form-dialog-title"
       fullWidth={fullWidth}
       maxWidth={maxWidth}
-      disableEscapeKeyDown
```

Comment:
Several of these lines were deleted. I'm guessing the original behavior was to save you from accidentally closing the modal while editing a particular field?